frontend/WikiContrib-Frontend/src/result.js

```diff
@@ -760,13 +760,16 @@ class QueryResult extends React.Component {
                 <Grid.Column width={2} />
               </Grid.Row>
               {this.state.activity !== undefined ? (
-                  <div className="activity_wrapper">
-                    <Activity
-                      date={this.state.activity}
-                      hash={this.state.query}
-                      username={this.state.current}
-                    />
-                  </div>
+                  <Grid.Row>
+                    <Grid.Column computer={2} mobile={1} tablet={1} />
+                    <Grid.Column computer={12} mobile={14} tablet={14} style={{ paddingTop: '2rem' }}>
```

Comment:
Top padding of 2rem seems too much here. You can reduce it to 1rem and then also add a same bottom padding of 1rem.